src/server.ts

```diff
@@ -17,7 +18,7 @@ export function configureApp(): express.Application {
   const app = express();
   app.use(express.json());
   app.use(cors());
-  app.use(bodyParser.json());
+  app.use(bodyParser.urlencoded({ extended: true }));
```

Comment:
You can actually leave them both
```suggestion
  app.use(bodyParser.json());
  app.use(bodyParser.urlencoded({ extended: true }));
```

Reply:
ok